test/unit/ngsiv2/plugins/bidirectional-plugin_test.js

```diff
@@ -323,6 +324,61 @@ describe('NGSI-v2 - Bidirectional data plugin', function () {
         });
     });
 
+    describe('When a new configGroup provisioning request arrives with bidirectional attributes', function () {
+        const provisionGroup = {
+            url: 'http://localhost:' + iotAgentConfig.server.port + '/iot/configGroups',
+            method: 'POST',
+            json: utils.readExampleFile('./test/unit/examples/groupProvisioningRequests/bidirectionalConfigGroup.json'),
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            }
+        };
+        const provisionDevice = {
+            url: 'http://localhost:' + iotAgentConfig.server.port + '/iot/devices',
+            method: 'POST',
+            json: utils.readExampleFile(
+                './test/unit/examples/deviceProvisioningRequests/provisionDeviceBidirectionalGroup.json'
+            ),
+            headers: {
+                'fiware-service': 'smartGondor',
+                'fiware-servicepath': '/gardens'
+            }
+        };
+
+        beforeEach(function () {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post(
+                    '/v2/subscriptions',
+                    utils.readExampleFile(
+                        './test/unit/ngsiv2/examples/subscriptionRequests/bidirectionalSubscriptionRequest.json'
+                    )
+                )
+                .reply(201, null, { Location: '/v2/subscriptions/51c0ac9ed714fb3b37d7d5a8' });
+
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post(
+                    '/v2/entities?options=upsert',
+                    utils.readExampleFile('./test/unit/ngsiv2/examples/contextRequests/createBidirectionalDevice.json')
+                )
+                .reply(204);
+        });
+        it('should subscribe to the modification of the combined attribute with all the variables', function (done) {
+            request(provisionGroup, function (error, response, body) {
+                request(provisionDevice, function (error, response, body) {
+                    should.not.exist(error);
+                    contextBrokerMock.done();
+                    done();
+                });
+            });
+        });
+    });
+
+    //This test will be removed if at the end /iot/services API (now Deprecated) is removed
```

Comment:
These marks are great :) However, I'd suggest to include the work "FIXME" in them:

```suggestion
    // FIXME: this test will be removed if at the end /iot/services API (now Deprecated) is removed
```

So it is easier to find pending things (e.g. grep) in the code base.

Reply:
Fixed in 6b87375